tools/docker/runner/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:18.04
```

Comment:
> I made a lot of comments; some may be a bit hard to understand in the way they are spread out in the code. The gist of it is:
> 
> * `prpl` user shouldn't be necessary since everything is run as root.
> * SHARED_SERVER shouldn't be needed since we can just use the SUDO_USER environment variable in the appropriate places.
> * It should be possible to start the daemons as part of the ENTRYPOINT so it starts in the background, and then use `docker exec` to start a shell.
> 
> However, all of these are major changes, and the other comments I made are minor nits, and the current approach works, and we shouldn't spend too much time on this infra, so I'm just approving. If you feel you should take into account any of my comments, feel free to do so, but you can also just merge.

I will fix whatever I can, and open a task for the rest so your (very good) comments don't go to waste.

Reply:
We can't do that since the builder image has things we don't need anymore in the runner image.
What we should do is create a common image for both and then use this one - but this will make it a bit of a hassle for users wanting only to use the builder or the runner.
The solution - just push our base image to the docker repository maybe?

Anyway for now I will skip this one, lets add it to the list of docker improvements.